spec/grape/endpoint_spec.rb

```diff
@@ -298,6 +298,56 @@ def app
       end
     end
 
+    context 'when params are not built with default class' do
+      it 'returns an object that corresponds with the params class - hash with indifferent access' do
+        subject.params do
+          build_with Grape::Extensions::ActiveSupport::HashWithIndifferentAccess::ParamBuilder
+
+          requires :first
```

Comment:
The actual parameters are not material for the tests, remove them. Make this as short as possible.